Dockerfile.provision-ip-discovery-agent

```diff
@@ -1,15 +1,45 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
-WORKDIR /go/src/github.com/openstack-k8s-operators/osp-director-operator
-COPY . .
-RUN go mod vendor
-RUN BIN_PATH=build/_output/cmd ./containers/provision_ip_discovery_agent/build-provision-ip-discovery-agent.sh
```

Comment:
hmm If I remember correctly this script does have some additional version injection stuff that wasn't done in here so not sure how to resolve the discrepancy and should probably get input from maintainer of that build script

Reply:
I have used CPaaS version of Dockerfile rather then logic from here, I can add it, will check with @abays if it's required.

Reply:
The `VERSION_OVERRIDE` in the build script `containers/provision_ip_discovery_agent/build-provision-ip-discovery-agent.sh` did not end up ever being used, if I recall correctly.  Is this the version injection to which you were referring, @yazug?

Reply:
yes, that is the part I was referring to, can we prune that out or the script out if this is sufficient?

Reply:
Yeah, just ignore that part of the bash script.  I borrowed it from somewhere else where the versioning stuff must have been needed, but then I didn't bother to pare it down for our version.